ddtrace/contrib/internal/rq/patch.py

```diff
@@ -13,21 +11,26 @@
 
 from ....ext import SpanKind
 from ....ext import SpanTypes
+from ....settings._config import _get_config
 from ... import trace_utils
 
 
 config._add(
     "rq",
     dict(
-        distributed_tracing_enabled=asbool(os.environ.get("DD_RQ_DISTRIBUTED_TRACING_ENABLED", True)),
+        distributed_tracing_enabled=asbool(
+            _get_config("DD_RQ_DISTRIBUTED_TRACING_ENABLED", None, report_telemetry=False)
```

Comment:
I think it makes sense to report this to telemetry so we can see if this config is used?
```suggestion
            _get_config("DD_RQ_DISTRIBUTED_TRACING_ENABLED", None)
```